src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -297,25 +290,21 @@ else if (key instanceof Map) {
             if (value instanceof Struct) {
                 final Element mapItemValueElement = addNode(doc, mapItemElement, "value");
                 processStruct(doc, mapItemValueElement, (Struct) value);
-            }
-            else if (value instanceof String) {
+            } else if (value instanceof String) {
                 addXmlTextNode(doc, mapItemElement, "value", value.toString());
-            }
-            else if (value instanceof byte[]) {
+            } else if (value instanceof byte[]) {
                 final Element mapItemValueElement = addNode(doc, mapItemElement, "value");
-                mapItemValueElement.setTextContent(Base64.getEncoder().encodeToString((byte[])value));
+                mapItemValueElement.setTextContent(Base64.getEncoder().encodeToString((byte[]) value));
```

Comment:
```suggestion
                mapItemValueElement.setTextContent(Base64.getEncoder().encodeToString((byte[])value));
```

Unrelated to this specific PR